fenix/app/nimbus.fml.yaml

```diff
@@ -86,7 +87,7 @@ import:
                 max-display-count: 5
               SURVEY:
                 priority: 55
-                max-display-count: 10
+                max-display-count: 1
```

Comment:
Should we create a new style, for this surface?  I guess not all surface will want to show only once, I think this will be a specific behaviour for fullscreen message.

Reply:
This was a decision made as a requirement to use this style.  And since we don't to show it more than once, I think 1 here is the correct value even for the homescreen version of the survey.  Thanks

Reply:
Thanks for the clarification 👍🏽